content/it/case-studies/box/index.html

```diff
@@ -0,0 +1,114 @@
+---
+title: Box Case Study
```

Comment:
nit: not translated

Reply:
(nor is the rest of the page)

Reply:
Yeah, I pulled it so the Box block appears in the case studies page. Being a copy of the English one, it does not change the rendering if not adding the box clock